### ISSUE TITLE
Area select: can't start in left and top margins

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#2900]
@JonFerraiolo 

Doing the area select test case (e.g., "VE 5"), I noticed I can't start an area selection in the white space in the left or top margins (e.g., where arrows are pointing in the screenshot):

![screenshot](http://i.imgur.com/s0n1B.png)

**Comment by JonFerraiolo** [comment]
Probably won't be able to fix this for M7, but leaving it open. The problem is that the white space on the left and right edge of the Button are a DIV in the Maqetta app's frame, whereas the Button is in the user's iframe. Right now the area select logic only tracks mouse down events in the user's iframe. Fixing this might be a bit of work.

**Comment by jhpedemonte** [comment]
I'm confused:  so there's white space that is a _border_ around the white space that is the _visual editor_?

If that is the case, shouldn't we differentiate between the two (i.e. not make them both white)?

**Comment by JonFerraiolo** [comment]
yes, we should differentiate the two areas. @jmockler and @jinsil were going to propose something. We can discuss at the next UI phone call.

**Comment by JonFerraiolo** [comment]
Pushing this to M8. 

There is now a gray gradient around the user canvas - only 1px of white that isn't part of the actual user's page, so we've taken care of one of the usability issues now.

Yes, we want to allow area select to extend outside of the user's iframe, but as you can imagine, that's must easier said than done. It's probably a couple of days of work, and somewhat risky.

**Comment by JonFerraiolo** [comment]
-> M8 Prio-Med

I've actually played with various fixes in this area, but I don't know if I'll have time to finish before M8.
